src/devtools/client/debugger/src/components/Editor/Breakpoints/Breakpoint.tsx

```diff
@@ -32,23 +50,19 @@ class Breakpoint extends PureComponent {
   }
 
   makeMarker() {
-    const bp = breakpointSvg.cloneNode(true);
+    const bp = breakpointSvg.cloneNode(true) as HTMLDivElement;
 
     bp.className = classnames("editor new-breakpoint", {
       "folding-enabled": features.codeFolding,
     });
 
     bp.onmousedown = this.onClick;
-    bp.onmouseenter = this.onMouseEnter;
```

Comment:
:speech_balloon: There _aren't_ any `onMouseEnter/Leave` methods in this component